src/workers/VolumeLoaderContext.ts

```diff
@@ -94,7 +98,13 @@ class SharedLoadWorkerHandle {
   private receiveMessage<T extends WorkerMsgType>({ data }: MessageEvent<WorkerResponse<T>>): void {
     if (data.responseResult === WorkerResponseResult.EVENT) {
       if (data.eventType === WorkerEventType.CHANNEL_LOAD) {
-        this.onChannelData?.(data);
+        if (this.onChannelData) {
+          if (this.throttleChannelData) {
+            throttle(() => (this.onChannelData ? this.onChannelData(data) : {}));
```

Comment:
why throttle here rather than e.g. when requests are sent, or before data is transferred from the worker?

Reply:
my thought here was that I mainly want to allow for a redraw in between onChannelDatas.  The throttle code lets them get automatically queued up with a time delay of my choosing.